page-archivo.php

```diff
@@ -73,14 +84,33 @@
     if ( $post->post_type === 'exposiciones' ) {
       $types = wp_get_post_terms($post->ID, 'tipo_de_exposicion', array('fields' => 'slugs'));
       $types = implode(' ', $types);
+
+      $meta = get_post_meta($post->ID);
+      $isExposicion = true;
+
     } else {
       $types = $post->post_type;
-    } 
+
+      $isExposicion = false;
+    }
 ?>
 
-      <article <?php post_class('filtered-content'); ?> id="post-<?php the_ID(); ?>" data-filter-type="<?php echo $types; ?>">
+      <article <?php post_class('archivo-post filtered-content'); ?> id="post-<?php the_ID(); ?>" data-filter-type="<?php echo $types; ?>">
         <a href="<?php the_permalink() ?>">
-          <h3 class="post-title"><?php the_title(); ?><h3>
+          <h2 class="post-title font-italic font-spaced"><?php the_title(); ?><h2>
+          <?php
+            if ($isExposicion) {
+              $start = $m = new \Moment\Moment('@' . $meta['_igv_start_date'][0]);
+              $end = $m = new \Moment\Moment('@' . $meta['_igv_end_date'][0]);
```

Comment:
yh i guess so. or is there a way/ it better to validate all exhibitions are saved with those dates